lib/Conversion/ATenToXTenPass.cpp

```diff
@@ -145,6 +144,53 @@ bool isLongestBranch(OpResult left, OpResult right) {
   return true;
 }
 
+// Tablegen adapter to check if attributes allow a conversion from `ReduceMean`
+// to `GlobalAveragePool`.
+bool isReduceMeanGlobalAveragePool2D(Value dims, Value keepdims) {
+
+  // Get defining operands
+  Torch::PrimListConstructOp dimsOp =
+      llvm::dyn_cast<Torch::PrimListConstructOp>(dims.getDefiningOp());
+  Torch::ConstantBoolOp keepdimsOp =
+      llvm::dyn_cast<Torch::ConstantBoolOp>(keepdims.getDefiningOp());
+
+  if (!dimsOp || !keepdimsOp)
+    return false;
+
+  SmallVector<APInt> axes;
+  for (Value operand : dimsOp.getOperands()) {
+    if (auto axesValue =
+            llvm::dyn_cast<Torch::ConstantIntOp>(operand.getDefiningOp()))
```

Comment:
Is it okay to ignore operands that are not `Torch::ConstantIntOp`?

Reply:
Yes, that is fine. They are always `torch.constant.int`. The arguments do not seem optional, but limited documentation as always.